README.md

```diff
@@ -2,58 +2,162 @@
 
 ## Introduction
 
-This plugin is meant to make it easy to quickstart development of new OBS plugins. It includes:
+The plugin template is meant to be used as a starting point for OBS Studio plugin development. It includes:
+
+* Boilerplate plugin source code
+* A CMake project file
+* GitHub Actions workflows and repository actions
+
+## Set Up
+
+The plugin project is set up using the included `buildspec.json` file - the following fields should be customized for an actual plugin:
+
+* `name`: The plugin name
+* `version`: The plugin version
+* `author`: Actual name or nickname of the plugin's author
+* `website`: URL of a website associated with the plugin
+* `email`: Contact email address associated with the plugin
+* `uuids`
+    * `macosPackage`: Unique (**!**) identifier for the macOS plugin plugin
+    * `macosInstaller`: Unique (**!**) identifier for the macOS plugin installer
+    * `windowsApp`: Unique (**!**) identifier for the Windows plugin installer
```

Comment:
I can add this, though I would like to think that "generating a valid UUID" is among the most basic skills somebody should already have when creating a plugin.

Reply:
I confess, I've never had to think about it. On *nix systems, I suppose one could easily use uuidgen, but I'm not sure what is used on Windows since I've never had to worry about it. Are these UUIDs expected to be v1, v4, or v5? Or does it not matter? Do we want to recommend a specific version?

Reply:
The default is random-based UUIDs (v4) to achieve a good-enough level of uniqueness. As long as we don't have a central plugin repository where plugins have to register there is also no way to check for collisions, but UUIDs as random identifiers are common enough (and easy enough to generate via `uuidgen` on Unix systems and `New-Guid` on Windows) for this use case.

(In theory the "bundle id" is actually better, but from what I could tell packages as well as ISS expect UUIDs in their files).